src/NzbDrone.Core/Localization/Core/en.json

```diff
@@ -34,6 +37,7 @@
   "AddNewSeriesSearchForMissingEpisodes": "Start search for missing episodes",
   "AddNotification": "Add Notification",
   "AddNotificationError": "Unable to add a new notification, please try again.",
+  "AddNotificationImplementation": "Add Notification - {implementation}",
```

Comment:
`Connection` instead of `Notification`

Reply:
Yeah, but I think `AddNotificationError` should be renamed changed to reference connections since that is the external facing name and `notifications` for the file/folder structure is from when it was really just notifications.